docs/view_components/welcome.rst

```diff
@@ -1,11 +1,47 @@
-Component title
----------------
+.. _welcome-label:
+
+Welcome page
+------------
+
+The welcome window serves as an orientation point for the user and let them know, in as brief words (and other audiovisual content) as possible, what to expect from the tool. It is also the launch point from where the user begins their energy systems simulation.
```

Comment:
If this *is* the launch point, this implies we can only launch it from here, which is not necessarily true

Reply:
you are right. The landing page view is the launching point and the welcome pop-up is only one of the view-components on this landing page. I will remove the last sentence.